templates/verify_login.txt

```diff
@@ -1,5 +1,9 @@
 {{t "New sign-in to Firefox"}}
-{{t "For added security, please confirm this sign-in to begin syncing with %(device)s." }}
+{{t "For added security, please confirm this sign-in to begin syncing with this device:" }}
+{{ device }}
+{{t "IP address: %(ip)s" }}
+{{ location }}
```

Comment:
Even with an if around this line, the extra line appears if there is no location information. Not sure what the solution is.

Let's leave this for now.